modules/translate.py

```diff
@@ -77,6 +77,9 @@ def modalCallback( self, modal, lang : Language_t ):
 
             if valid:
                 self.context.read_write.writeTextFile( file['filename'], text )
+
+                current_language : self.Language_t = self.getCurrentLanguage()
```

Comment:
```suggestion
                current_language : Translate.Language_t = self.getCurrentLanguage()
```

Reply:
self. works because ```Translate``` is a nested type, to reference it from other classes ```Translate.Language_t``` is required.
using self keeps it concise. Both are valid though.